### PR TITLE
Adding Telekom + Lufthansa shared credential backends

### PR DESCRIPTION
**Pull request by jumoru**
Lufthansa: [miles-and-more.com](miles-and-more.com) is the loyalty program of Lufthansa, you can log into both sites using the same credentials (Lufthansa-ID)

Telekom: You have to register on the [telekom-dienste.de](https://meinkonto.telekom-dienste.de/telekom/account/registration/registration.xhtml) domain, but when logging into their services (cloud, email,  customer center, ...) you're always redirected [accounts.login.idm.telekom.com](accounts.login.idm.telekom.com). Example: Click on the Login button on their site for their [customer center](https://www.telekom.de/kundencenter/startseite).

### CODE REVIEW

File: quirks/websites-with-shared-credential-backends.json [hunk 1]
```diff
@@ -218,4 +218,12 @@
         "lookmark.io",
         "lookmark.link"
     ]
+    [
```
Comment (bradcush) [inline quirks/websites-with-shared-credential-backends.json:221]: Some commas are missing to separate the two new lists that you've added

Comment (MaivKhanh) [reply quirks/websites-with-shared-credential-backends.json:221]: 1

Comment (jumoru) [reply quirks/websites-with-shared-credential-backends.json:221]: Thanks for alerting me to this, I've added the missing commas

File: quirks/websites-with-shared-credential-backends.json [hunk 2]
```diff
@@ -217,5 +217,13 @@
     [
         "lookmark.io",
         "lookmark.link"
+    ],
+    [
+        "lufthansa.com",
+        "miles-and-more.com"
+    ],
```
Comment (rmondello) [inline quirks/websites-with-shared-credential-backends.json:224]: And just to be sure, there are login pages on both of these domains that can be used to sign in with the same credentials?

Comment (DerJuulsn) [reply quirks/websites-with-shared-credential-backends.json:224]: Can confirm, use it on my own.

**Comment by jumoru** [comment]
Yes, you can log into both using the same credentials. Miles and More is the loyalty program of Lufthansa. And sorry for not providing some explanatory comments to begin with.

**Comment by rmondello** [comment]
Great! @jumoru, can you update the PR to allow it to merge cleanly?

**Comment by jumoru** [comment]
Of course. I've updated the PR and the initial comment with some explanations behind the proposed changes. Thanks @rmondello for your feedback and assistance.

**Comment by rmondello** [comment]
Thanks for the explanations! They helped a lot.

**Pull request merged**